### PR TITLE
Add Wilander as editor and move Barth from Authors to Acknowledgements.

### PR DESCRIPTION
**Pull request by johnwilander**
Ugh. This process is killing me.

### CODE REVIEW

**Comment by johnwilander** [comment]
I don't understand the failed check. Do you, Mike?

File: draft-ietf-httpbis-rfc6265bis.md [hunk 1]
```diff
@@ -12,17 +12,18 @@ pi: [toc, tocindent, sortrefs, symrefs, strict, compact, comments, inline]
 stand_alone: yes #_
 
 author:
--
-  ins: A. Barth
-  name: Adam Barth
-  organization: Google, Inc
-  uri: https://www.adambarth.com/
 -
```
Comment (LPardue) [inline draft-ietf-httpbis-rfc6265bis.md]:
```suggestion
  -
```
identation problem perhaps

Comment (johnwilander) [reply draft-ietf-httpbis-rfc6265bis.md]: This is exactly the thing I tried to address. Maybe I failed.

**Comment by MikeBishop** [comment]
> I don't understand the failed check. Do you, Mike?

I'm probably not the Mike you meant, but it appears to be about references in `-priorities`.  Nothing to do with this change.

> - No link definition for link ID '-1,6' found on line 855
> - No link definition for link ID '0,7' found on line 855

**Pull request closed without merging**